bc-react/src/stores/UserStore.js

```diff
@@ -36,6 +35,12 @@ class UserStore extends EventEmitter{
         this.emit('login-fail')
         break
       }
+      case("DELETE_USER"):{
```

Comment:
@rachelruderman I don't think you do anything with the userstore for delete, only the admin store